management/management_api/endpoints/endpoints.py

```diff
@@ -69,7 +72,7 @@ def on_patch(self, req, resp, tenant_name, endpoint_name):
 
         message = 'Endpoint {} patched successfully. New values: {}\n'.format(endpoint_url, body)
         resp.status = falcon.HTTP_200
-        resp.body = message
+        resp.body = json.dumps({'status': 'PATCHED', 'data': {'url': endpoint_url, 'values': body}})
         logger.info(message)
```

Comment:
In some functions we log on this level (`on_patch, on_post, on_delete`) and in some we do it inside utils function e.g. delete_endpoint. Maybe we could extract logging from utils functions so it will be on the same level as falcon returns?